chart/templates/jobs/metrics-collector/job.yaml

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2022 The HuggingFace Authors.
+
+{{- if .Values.images.jobs.cacheMaintenance }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  labels: {{ include "labels.metricsCollector" . | nindent 4 }}
+  name: "{{ include "name" . }}-job-metrics-collector"
+  namespace: {{ .Release.Namespace }}
+spec:
+  schedule: "*/3 * * * *"
```

Comment:
I think that the schedule could be a variable (we might want a different value in prod and dev)

Reply:
I added a config for it in values.yaml